README.md

```diff
@@ -46,3 +46,8 @@ and install ABRT can be found in [INSTALL.md](INSTALL.md)
 * [D-Bus](https://www.freedesktop.org/wiki/Software/dbus/)
 * [SELinux](https://github.com/SELinuxProject/selinux/wiki)
 * [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
+
+
+### Translations
+
+[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/287x66-grey.png)](https://translate.fedoraproject.org/engage/abrt/?utm_source=widget)
```

Comment:
This is wrong.

```
[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/abrt/svg-badge.svg)](https://translate.fedoraproject.org/engage/abrt/?utm_source=widget)
```

And maybe even without the tracking:

```
[![Translation status](https://translate.fedoraproject.org/widgets/abrt/-/abrt/svg-badge.svg)](https://translate.fedoraproject.org/engage/abrt/)
```

Reply:
Also maybe put it on top? You now have to scroll *all* the way down to see the badge.

Reply:
These heading would also have to go away.